<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDisplayOptions</class>
 <widget class="QDialog" name="DialogDisplayOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QComboBox" name="cmbNormaliseOverAllSpecies">
     <item>
      <property name="text">
       <string>maximum of each species</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>maximum over all species</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cmbNormaliseOverAllTimepoints">
     <item>
      <property name="text">
       <string>at each timepoint</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>over all timepoints</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTreeWidget" name="listSpecies">
     <column>
      <property name="text">
       <string>Species to display</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="chkShowMinMaxRanges">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Show min/max ranges on plot</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblNormalise">
     <property name="text">
      <string>Normalise concentration image colour intensity to:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listSpecies</tabstop>
  <tabstop>chkShowMinMaxRanges</tabstop>
  <tabstop>cmbNormaliseOverAllSpecies</tabstop>
  <tabstop>cmbNormaliseOverAllTimepoints</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
